bleak/backends/bluezdbus/client.py

```diff
@@ -340,6 +340,8 @@ async def connect(self, **kwargs) -> bool:
             # Get all services. This means making the actual connection.
             await self.get_services()
 
+            await self._acquire_mtu()
```

Comment:
you're not exposing it ;-) you're calling it

Reply:
Reworded the commit 😆